### DISCUSSION TITLE
流日的宫位计算出错，期待更新

### DISCUSSION THREAD

**Discussion by yushinelee** [SylarLong/iztro#119] [Q&A]
流日的宫位计算出错，期待更新

**Comment by SylarLong** [comment]
Ref. https://github.com/SylarLong/iztro/issues/91

你好 @yushinelee ，请问可以留下你的参数么？可能是UT没有覆盖到某些特殊情况，如果有参数的话帮助会很大，比如输入的日期，日期类型和时辰。

**Reply by SylarLong** [reply]
由于报告这过长时间无响应，决定关闭
